docs/advanced_configuration.md

```diff
@@ -0,0 +1,83 @@
+## Overview
+
+If you have not set up the SDK yet, follow the [in-app setup instructions][1] or refer to the [React Native RUM setup documentation][2].
+
+## Manual instrumentation
+
+If automatic instrumentation doesn't suit your needs, you can manually create RUM Events and Logs:
+
+{{< code-block lang="javascript" filename="Initialization Snippet" collapsible="true" >}}
```

Comment:
This makes this file impossible to read on Github.

I think we need to find a good way to leverage the features that the docs website can offer, while still maintaining a coherent documentation here.

I see a few ways of doing that:
- moving this file to the documentation repository
  - but this will make it a bit harder for us to maintain
- moving this file in a directory that clearly indicates that it is intended for the docs website, and have a version for github
  -  for instance by having `docs` and `internal_docs` folders in the repo

What do you think?

If we can't find a solution quickly, I believe that adding the collapsible on this block but making it unreadable in GitHub is not a worthy tradeoff, so I'd rather have it always visible on the docs website and readable here by changing 

```
{{< code-block lang="javascript" filename="Initialization Snippet" collapsible="true" >}}
```

into:

` ``` `

Reply:
Agreed, I thought users may have wanted to use the Copy button for the code snippet, but we don't have to use it!